generative/smrd/models/models.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch.nn as nn
+from mutils import get_sigmas
+from .layers import *
+from .normalization import get_normalization
+
+
+class UNet(nn.Module):
```

Comment:
I think we may need to use the components from the MONAI repo.

Reply:
Did you mean [this](https://docs.monai.io/en/1.3.0/_modules/monai/networks/nets/unet.html) one?

Reply:
Yes, do we have specific reason to add this pretrained model otherwise we can just ensure functionality level works?
What do you think? cc @ericspod

Reply:
It would be best wherever possible to use MONAI components. We have our own UNet class but is structured differently from this one, and we do have other blocks and layers that could be substituted for those defined here. If possible please use those in MONAI unless it causes significant headaches converting the weight names in your pretrained data.